Exercise 3 : REST API

```diff
@@ -0,0 +1,100 @@
+Apex Class
+
+public class WeatherService {
+   
+    public class Main {
+        public Decimal temp;
+    }
+
+   
+    public class Weather {
+        public String description;
+        public String icon;
+    }
+
+    @AuraEnabled(cacheable=true)
+    public static WeatherResponse getWeather(String city) {
+        String apiKey = '536624f44f525cdabe70892cc8213f29'; 
```

Comment:
Don't hardcode sensitive data in your code. Salesforce provides secure ways to store and retrieve these: https://developer.salesforce.com/docs/atlas.en-us.secure_coding_guide.meta/secure_coding_guide/secure_coding_storing_sensitive_data.htm